clients/admin-ui/src/features/system/privacy-declarations/PrivacyDeclarationForm.tsx

```diff
@@ -147,25 +164,6 @@ export const PrivacyDeclarationFormComponents = ({
   );
 };
 
-/**
```

Comment:
moved this into `hooks.ts` so that this file can be totally independent of redux